src/createServer.js

```diff
@@ -1,9 +1,38 @@
 /* eslint-disable no-console */
 'use strict';
 
+const http = require('http');
+
 function createServer() {
-  /* Write your code here */
-  // Return instance of http.Server class
+  const server = http.createServer((req, res) => {
+    const params = new URL(`http://${req.headers.host}${req.url}`);
+    const parts = params.pathname.split('/').filter((el) => el !== '');
+
+    const queries = [];
+
+    for (const qury of params.searchParams) {
+      const nameQuery = qury[0];
+      const targetCase = qury[1];
+
+      queries.push([nameQuery, targetCase]);
```

Comment:
If `queries` is only used to create `query` why not create and populate `query` in this for achieving a one-pass solution instead of making it two-pass?